### PR TITLE
CWE-862

### PR DESCRIPTION
**Pull request by A-R05**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-2694

### Change description ###

Removed Suppression for CWE-862. Ran dependancyCheckAggregate with no issues 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

**Review by paulridout — APPROVED** [review]
Changes look OK.

**Review by MSancaktutar — APPROVED** [review]
lgtm

**Review by MSancaktutar — APPROVED** [review]

File: dependency-check-suppressions.xml [hunk 1]
```diff
@@ -47,13 +47,21 @@
 	</suppress>
 
 	<suppress until="2022-06-25">
-    		<notes><![CDATA[
-   			file name: spring-security-core-5.4.7.jar
-   		]]></notes>
-    		<packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-core@.*$</packageUrl>
-    		<vulnerabilityName>CWE-862: Missing Authorization</vulnerabilityName>
-  	</suppress>
+		<notes><![CDATA[
```
Comment (rajaram-nagarajan) [inline dependency-check-suppressions.xml:50]: These unwanted suppressions needs to be removed (I think you might have added it to keep the build green, but these were fixed last week)

Comment (rajaram-nagarajan) [inline dependency-check-suppressions.xml:50]: These unwanted suppressions needs to be removed (I think you might have added it to keep the build green, but these were fixed last week)

File: build.gradle [hunk 2]
```diff
@@ -49,7 +49,7 @@ dependencyCheck {
 ext['elasticsearch.version'] = '7.17.1'
 ext['javax-validation.version'] = '2.0.1.Final'
 ext['hibernate-validator.version'] = '6.0.20.Final'
-ext['spring-security.version'] = '5.4.7'
```
Comment (rajaram-nagarajan) [inline build.gradle:52]: use 5.6.1 version

Comment (rajaram-nagarajan) [inline build.gradle:52]: use 5.6.1 version

File: build.gradle [hunk 3]
```diff
@@ -133,6 +133,7 @@ allprojects {
         // CVE-2021-42550
         compile group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.10'
         compile group: 'ch.qos.logback', name: 'logback-core', version: '1.2.10'
+        compile group: 'org.springframework.security', name: 'spring-security-core', version: '5.7.1'
```
Comment (rajaram-nagarajan) [inline build.gradle:136]: do we need this?

File: build.gradle [hunk 4]
```diff
@@ -215,13 +216,15 @@ subprojects { subproject ->
             exclude group: 'com.google.guava', module: 'guava'
         }
 
-        compile "org.springframework.security:spring-security-web"
-        compile "org.springframework.security:spring-security-config"
+        compile "org.springframework.security:spring-security-web:5.7.1"
```
Comment (rajaram-nagarajan) [inline build.gradle:219]: we might not need to mention versions here.

Comment (rajaram-nagarajan) [inline build.gradle:219]: we might not need to mention versions here.

File: application/src/test/resources/mappings/oauth-jwk-set.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
+{
+  "request": {
```
Comment (rajaram-nagarajan) [inline application/src/test/resources/mappings/oauth-jwk-set.json:2]: can you briefly explain why do we need this change?

Comment (rajaram-nagarajan) [inline application/src/test/resources/mappings/oauth-jwk-set.json:2]: can you briefly explain why do we need this change?

**Pull request closed without merging**